pwiz_tools/Skyline/Util/FormulaWithMassModification.cs

```diff
@@ -0,0 +1,628 @@
+﻿/*
+ * Original author: Brian Pratt <bspratt .at. protein.ms>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
+ *
+ * Copyright 2023 University of Washington - Seattle, WA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using pwiz.Common.Chemistry;
+using pwiz.Common.Collections;
+
+namespace pwiz.Skyline.Util
+{
+    /// <summary>
+    /// Based on Formula class in this same namespace, this adds the ability to describe a mass
+    /// offset of unknown origin.
+    /// </summary>
+    public class FormulaWithMassModification : IDictionary<string, int>, IComparable
```

Comment:
I don't think this source file is mentioned in Skyline.csproj

Reply:
Yeah, that's cruft. I'll delete it, thanks